driver/src/main/scala/org/mongodb/scala/model/package.scala

```diff
@@ -81,6 +81,25 @@ package object model {
   }
 
   /**
+   * The default options for a collection to apply on the creation of indexes.
+   *
+   * @Note Requires MongoDB 3.2 or greater
+   * @see [[http://docs.mongodb.org/manual/reference/command/createIndexes Index options
+   * @since 1.1
+   */
+  type IndexOptionDefaults = com.mongodb.client.model.IndexOptionDefaults
```

Comment:
Its no good my OCD wont allow it - will pick and merge with the InsertOne commit post review.